backend/elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/index/impl/ElasticsearchIndexWorkPlan.java

```diff
@@ -40,12 +38,11 @@
 
 	private final List<ElasticsearchWork<?>> works = new ArrayList<>();
 
-	ElasticsearchIndexWorkPlan(ElasticsearchWorkFactory factory, MultiTenancyStrategy multiTenancyStrategy,
```

Comment:
Could you please preserve the original order of parameters? The factory was before the multi-tenancy strategy, because it's more important.

Reply:
sure

Reply:
Do we need to keep the same order, factory < multi-tenancy strategy, in `ElasticsearchIndexWorkExecutor` and `ElasticsearchIndexDocumentWorkExecutor` too?